src/small-search-bar.css

```diff
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: 'Avenir book';
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: 'Avenir heavy';
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.page__form_small {
+  margin-top: 20px;
+}
+
+.form_small {
+  display: flex;
+  align-items: center;
+  height: 40px;
```

Comment:
I think that the element should have height for example input or label. Don't fix container size (if there is no such a requirement) rule is broken here.